PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -4,6 +4,7 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        Console.WriteLine("Welcome to Princess Bride Trivia");
```

Comment:
Are you sure you meant to leave this line?
```suggestion
        Console.WriteLine("Welcome to Princess Bride Game");
```